code/modules/surgery/organs/augments_arms.dm

```diff
@@ -449,6 +454,76 @@
 	action_icon = list(/datum/action/item_action/organ_action/toggle = 'icons/obj/janitor.dmi')
 	action_icon_state = list(/datum/action/item_action/organ_action/toggle = "advmop")
 
+// Shell launch system, an arm mounted single-shot shotgun that comes out of your arm
+
+/obj/item/gun/projectile/revolver/doublebarrel/shell_launcher
```

Comment:
Shouldn't it have nodrop?

Reply:
Qwerty on mobile here, antidrop is given to anything put into an arm implant